yarn-project/acir-simulator/src/index.test.ts

```diff
@@ -95,12 +95,12 @@ describe('ACIR simulator', () => {
 
     function buildNote(amount: bigint, owner: NoirPoint, isDummy = false) {
       return [
-        new Fr(amount),
+        new Fr(isDummy ? 1n : 0n),
```

Comment:
I know this doesn't matter cause it's just for testing and a developer can do whatever they want XD But maybe the flag can be `isReal` so that a dummy note will be all zeros.